AsyncDisplayKit/ASTableView.mm

```diff
@@ -806,6 +806,7 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
     cell.node = node;
     cell.backgroundColor = node.backgroundColor;
     cell.selectionStyle = node.selectionStyle;
+    cell.selectedBackgroundView = node.selectedBackgroundView;
```

Comment:
Would you be willing to add the same code into `-[ASCollectionView collectionView:cellForItemAtIndexPath:]`?